components/brave_webtorrent/extension/manifest.json

```diff
@@ -20,6 +20,6 @@
       "listen": "*:*"
     }
   },
-  "content_security_policy": "default-src 'self'; connect-src 'self' http: https:; font-src 'self' data:; script-src 'self'; media-src 'self' http://127.0.0.1:*; form-action 'none'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; frame-src 'self' http://127.0.0.1:*;",
+  "content_security_policy": "default-src 'self'; connect-src 'self' http: https:; font-src 'self' data:; script-src 'self'; media-src 'self' http://127.0.0.1:*; form-action 'none'; style-src 'self' 'unsafe-inline'; img-src 'self' data: http://127.0.0.1:*; object-src 'self' http://127.0.0.1:*; frame-src 'self' http://127.0.0.1:*;",
```

Comment:
Two changes here:

- Images need to load from `http://127.0.0.1:*`
- We allow use of the `object` tag , loading from `'self'` and `http://127.0.0.1:*`